pkg/agent/bgp/interface.go

```diff
@@ -59,6 +62,18 @@ type GlobalConfig struct {
 	ListenPort int32
 }
 
+type LogLevel int32
+
+const (
+	Panic LogLevel = 0
+	Fatal LogLevel = 1
+	Error LogLevel = 2
+	Warn  LogLevel = 3
+	Info  LogLevel = 4
+	Debug LogLevel = 5
+	Trace LogLevel = 6
+)
```

Comment:
Should we use `iota`?

I would recommend limiting the number of log levels rather than having a verbatim copy of the gobgp ones. Maybe we could just have `error`, `info`, `debug` and `trace`?
- `error` would enable panic, fatal and error
- `info` would also enable warn and info
- `debug` would also enable `debug`
- `trace` would also enable `trace`